### DISCUSSION TITLE
New Interface: Repeater of Inputs Saved as CSV

### DISCUSSION THREAD

**Discussion by keesvanbemmel** [directus/directus#9604] [Feature Requests]
Hi there!

Just wondering, what would be the best way of storing 1 or more strings in a collection. I have an Item called shipments that can have 1 or more tracking URLs. Is there a way to store them separately in one field?

Cheers!

Kees.

**Comment by joselcvarela** [comment]
Hello @keesvanbemmel ,
In new field flow you have `Tags` type. I believe that should the best one.

**Reply by keesvanbemmel** [reply]
Makes sense, just wouldn't have thought of it to use for tracking URLs. Will give it a try!

**Reply by benhaynes** [reply]
The `tags` interface is the best for what we have now... though I would love to see an "input repeater" style CSV interface!! Maybe I'll add a Feature Request for that. :)

**Reply by benhaynes** [reply]
Actually, I just turned this into that feature request to keep the "noise" down. 🙂

**Reply by benhaynes** [reply]
Thoughts on something like this?

![Screen Shot 2021-11-08 at 10 54 48 AM](https://user-images.githubusercontent.com/522079/140774543-0e7e356f-32bc-41e0-9645-ca8ab5ae973d.png)

**Comment by keesvanbemmel** [comment]
Thanks @benhaynes!

**Reply by keesvanbemmel** [reply]
Actually, that looks exactly like the thing I was looking for

**Reply by benhaynes** [reply]
Nice! Would probably be pretty easy to create, but even so, our core team won't have time for a while due to the v9 release. Potential options:

* You could try to create a PR yourself
* You could try to convince someone in the community to contribute it
* You could fund this on IssueHunt to motivate a contributor (I'll throw in $20 too)
* You could sponsor this for the core team to build
* You could wait for this to get done by the core team (no time estimates here)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
